src/main/java/com/google/devtools/build/lib/bazel/rules/python/python_stub_template.txt

```diff
@@ -2,6 +2,10 @@
 
 from __future__ import print_function
 
+import sys
+
+del sys.path[0]
```

Comment:
Is this an incompatible change?

Reply:
No, this is the crux of the fix to the linked bug. It affects only the stub Python script, not the payload process, whose path is still determined by the `python_path_entries` logic in `main()`.

Reply:
Thanks!